sonar-scanner-engine/src/main/java/org/sonar/batch/bootstrap/ExtensionUtils.java

```diff
@@ -36,9 +37,10 @@ public static boolean isInstantiationStrategy(Object extension, String strategy)
     }
     return InstantiationStrategy.PER_PROJECT.equals(strategy);
   }
-
-  public static boolean isBatchSide(Object extension) {
-    return AnnotationUtils.getAnnotation(extension, BatchSide.class) != null;
+  
+  public static boolean isScannerSide(Object extension) {
+    return AnnotationUtils.getAnnotation(extension, BatchSide.class) != null ||
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "BatchSide"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)